pyuvdata/uvcal/tests/test_fhd_cal.py

```diff
@@ -77,24 +85,25 @@ def test_read_fhdcal_raw_write_read_calfits(raw, tmp_path):
         raw=raw,
         read_data=False,
     )
-    calfits_cal2 = calfits_cal.copy(metadata_only=True)
+
+    fhd_cal2 = fhd_cal_full.copy(metadata_only=True)
 
     # this file set has a mismatch in Nsources between the cal file & settings
     # file for some reason. I think it's just an issue with the files chosen
-    assert fhd_cal.Nsources != calfits_cal2.Nsources
-    fhd_cal.Nsources = calfits_cal2.Nsources
+    assert fhd_cal.Nsources != fhd_cal2.Nsources
+    fhd_cal.Nsources = fhd_cal2.Nsources
 
     # there is a loss in precision for float auto scale values in the
     # settings file vs the cal file
     assert (
         fhd_cal.extra_keywords["autoscal".upper()]
-        != calfits_cal2.extra_keywords["autoscal".upper()]
+        != fhd_cal2.extra_keywords["autoscal".upper()]
```

Comment:
This is a nit-pick, but are the approximately true? Like would they pass a `numpy.isclose` test? If so I might advocate for doing that before overwriting the values so the object equality will pass.